mender-update/standalone/standalone.cpp

```diff
@@ -417,19 +422,75 @@ ResultAndError DoInstallStates(
 		return {Result::FailedNothingDone, payload.error()};
 	}
 
+	auto default_paths {main_context.GetConfig().paths};
+	events::EventLoop loop;
+	auto script_runner {executor::ScriptRunner(
+		loop,
+		chrono::seconds {main_context.GetConfig().state_script_timeout_seconds},
+		default_paths.GetArtScriptsPath(),
+		default_paths.GetRootfsScriptsPath())};
+
+
+	// Download Enter
+	{
+		auto err = script_runner.RunScripts(executor::State::Download, executor::Action::Enter);
+		if (err != error::NoError) {
+			auto leave_error {
+				script_runner.RunScripts(executor::State::Download, executor::Action::Error)};
```

Comment:
Nit pick,

I think here (and below) the `leave_` prefix is confusing, is should be `download_error` as this is an error when executing the `Error` action of the script, not the `Leave`.

Reply:
Thanks! Good idea, this was actually bothering me, but I couldn't think of anything better.